data/init/dfhack.keybindings.init

```diff
@@ -40,14 +40,15 @@ keybinding add Ctrl-Shift-K gui/cp437-table
 # clean the selected tile of blood etc
 #keybinding add Ctrl-C spotclean
 
-# destroy the selected item
-#keybinding add Ctrl-K@dwarfmode autodump-destroy-item
+# teleport all unforbidden items marked for dumping to the tile under the keyboard cursor
+keybinding add Ctrl-Alt-D autodump
```

Comment:
What's the conflict with Ctrl-K?

Reply:
Nothing, at the moment. I'm just trying to set a pattern that will result in no surprises. This is what I'm tentatively sticking to:  Modal tool windows can use bare keys, Ctrl keys, and alt keys, regular tool windows (ZScreens) can use Ctrl keys and alt keys, invisible keybindings can use ctrl-shift keys and ctrl-alt keys.

Reply:
There is a bit more risk that using more modifiers will conflict with system shortcuts (e.g. ctrl-alt-T opens a terminal under either Ubuntu or GNOME), as opposed to our own "tried-and-true" shortcuts.

Actually, `ctrl-alt-D` on Ubuntu/GNOME _hides all windows_, equivalent to Windows+D on Windows. I'm pretty sure that's a default, at least.

Reply:
it's generally risky to use ctrl-alt keys for any application purpose because those are widely considered reserved to system software

also, the msft UI guidelines say this:
> Avoid ALT+letter combinations because they may conflict with access keys. In addition, the system uses many specific key combinations for specialized input; for example, ALT+~ invokes an input editor for the Japanese language.
> Avoid CTRL+ALT combinations because the system interprets this combination in some language versions as an ALTGR key, which generates alphanumeric characters.

Reply:
reverting autodump hotkey changes, though I might reinstate existing hotkeys for `quicksave` and `quickfort`